nixos/modules/services/network-filesystems/drbd.nix

```diff
@@ -47,18 +47,33 @@ let cfg = config.services.drbd; in
         options drbd usermode_helper=/run/current-system/sw/bin/drbdadm
       '';
 
-    environment.etc.drbd.conf =
+    environment.etc."drbd.conf" =
       { source = pkgs.writeText "drbd.conf" cfg.config; };
 
     systemd.services.drbd = {
+      description = "Distributed Replicated Block Device";
       after = [ "systemd-udev.settle.service" "network.target" ];
       wants = [ "systemd-udev.settle.service" ];
       wantedBy = [ "multi-user.target" ];
+      serviceConfig = {
+        Type = "oneshot";
+        RemainAfterExit = "yes";
+      };
+      preStart = ''
+        # Check the configuration file for syntax errors
+        ${pkgs.drbd-utils}/bin/drbdadm sh-nop
```

Comment:
Would `drbdadm` allow us to run this inside the sandbox and present build time errors?

Reply:
Maybe! But I'd prefer to not delay this to figure that out.